### PR TITLE
[MIG] document_page_procedure: Migration to 14.0

### PR DESCRIPTION
**Pull request by dzungtran89**
Dependencies:
- document_page: https://github.com/OCA/knowledge/pull/283
- mgmtsystem: https://github.com/OCA/management-system/pull/365

### CODE REVIEW

**Comment by YvanDotet** [comment]
Hello
Is it ready to merge in the 14.0 branch ?
Thank you.

File: test-requirements.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+odoo14-addon-knowledge @git+https://github.com/dzungtran89/knowledge@14.0-mig-knowledge#subdirectory=setup/knowledge
+odoo14-addon-document-page @git+https://github.com/dzungtran89/knowledge@14.0-mig-document_page#subdirectory=setup/document_page
+odoo14-addon-mgmtsystem @git+https://github.com/dzungtran89/management-system@14.0-mig-mgmtsystem#subdirectory=setup/mgmtsystem
```
Comment (sebalix) [inline test-requirements.txt]: To clean up, these dependencies have been merged

**Review by sebalix — COMMENTED** [review]
LG, `test-requirements.txt` to clean up

**Review by simahawk — APPROVED** [review]

**Review by sebalix — APPROVED** [review]

**Pull request merged**